README.md

```diff
@@ -100,6 +100,22 @@ For extra security, you can disable certain Redis commands (this is especially i
       - CONFIG
       - SHUTDOWN
 
+Enable replication.
+
+    redis_replication: false
```

Comment:
Shouldn't it make sense to set this to true instead when ones wants to enable replication?
Also, perhaps you could include text that this doesn't enable Redis Sentinel (ref to the docs), but simply makes a master/slave setup.